plugins/defaultplugins/l2plugin/bd_config.go

```diff
@@ -319,12 +319,29 @@ func (plugin *BDConfigurator) ResolveCreatedInterface(interfaceName string, inte
 	return nil
 }
 
-// ResolveDeletedInterface does nothing
-func (plugin *BDConfigurator) ResolveDeletedInterface(interfaceName string) {
+// ResolveDeletedInterface removes interface from bridge domain operational status
```

Comment:
We need to talk how operational state for BDs work, let's have webex. Let's not merge this until this happens...

Reply:
well then, but it works the same way as for interfaces